src/TeamsDriverStationDict.elm

```diff
@@ -0,0 +1,16 @@
+module TeamsDriverStationDict exposing (..)
```

Comment:
Don't use a wildcard (even when you really do want to expose everything) - it's much easier to expose more symbols later then to hide symbols that were previously public.